src/BuildingRegistry/Building/Events/BuildingUnitWasCorrectedFromRetiredToRealized.cs

```diff
@@ -21,33 +21,38 @@ public class BuildingUnitWasCorrectedFromRetiredToRealized : IBuildingEvent
         public int BuildingUnitPersistentLocalId { get; }
 
         [EventPropertyDescription("Extended WKB-voorstelling van de gebouweenheidpositie (Hexadecimale notatie).")]
-        public string? DerivedExtendedWkbGeometry { get; }
+        public string DerivedExtendedWkbGeometry { get; }
```

Comment:
If the position didn't change, why include it in the event? Is this changed to fix flaky tests?